automation_tools/satellite6/upgrade/tasks.py

```diff
@@ -402,10 +402,13 @@ def sync_capsule_tools_repos_to_upgrade(admin_password=None):
     run(initials + 'content-view publish --name {0} '
         '--organization {1}'.format(cv_name, org))
     # promote cv
-    lc_env_id = str(
-        run(initials + 'lifecycle-environment list '
-            '--organization {0} | grep {1}'.format(org, env_name))).split(
-                '|')[0].strip()
+    lc_list = str(
+        run(initials + '--output csv lifecycle-environment list '
```

Comment:
You can also pass the name to the list command which will help filter even more the results:

```
[root@qe-sat6-rhel7 ~]# hammer lifecycle-environment list --organization-id 1 --name "Library"
[Foreman] Password for admin: 
---|---------|------
ID | NAME    | PRIOR
---|---------|------
1  | Library |      
---|---------|------
```